xpk/xpk.py

```diff
@@ -40,8 +40,8 @@
 import string
 import subprocess
 import sys
-import tempfile
 import time
+import utils
```

Comment:
nit: curious if you think xpk_utils class would be better since it is more specific?

Reply:
Good point, added